i18n/nl.json

```diff
@@ -4217,7 +4217,7 @@
   "status_dropdown.set_online": "Online",
   "status_dropdown.set_ooo": "Out Of Office",
   "status_dropdown.set_ooo.extra": "Automatische Antwoorden zijn ingeschakeld",
-  "status_dropdown.set_your_status": "Stel je status in",
+  "status_dropdown.set_your_status": "Status",
```

Comment:
Wondering if this is an intentional change?

Reply:
I changed the base, this change was due to that. Can you check again?